addons/website_quote/views/website_quotation.xml

```diff
@@ -692,7 +692,12 @@
           <t t-foreach="quotations" t-as="quotation">
               <tr>
                   <td>
-                      <a t-attf-href="/quote/#{quotation.id}?#{keep_query()}"><t t-esc="quotation.name"/></a>
+                      <t t-if="quotation.access_token">
```

Comment:
I was thinking that we could only display the tokenized link if the current user is a portal user an is not in the same company as the quote's customer (because this what causes the access error); if we do something more specific like
```xml
<t t-if="quotation.access_token and quotation.partner_id.commercial_partner_id != request.env.user_id.partner_id.commercial_partner_id and request.env.user.share">```
we don't lose as much traceability in the chatter (i.e. all internal users and partners of the same company are correctly logged when they view the quote online)